package-lock.json

```
@@ -93,8 +93,8 @@
         "yargs": "17.2.1"
       },
       "engines": {
-        "node": ">=16.17.0 <17",
-        "npm": ">=8.11.0 <9"
+        "node": ">=18.13.0 <19",
+        "npm": ">=9.6.5 <10"
```

18.13.0のnpmのデフォルトバージョンは8.19.3でした。
今の最新のNode18についてくるnpmは9でした。
なのでこうがいいかも？
```suggestion
        "npm": ">=8.19.3 <10"
```